### ISSUE TITLE
windows 10 rtx2080ti gpu build   test CIFAR  

### ISSUE THREAD

**Issue by jaiminlee** [pjreddie/darknet#2229]
cuda_convert_f32_to_f16() : line: 138   error 

 What is it?

D:\YOLOv4\darknet-master\build\darknet\x64\data\cifar>darknet classifier train cifar.data cifarsmall.cfg
 CUDA-version: 10010 (11000), cuDNN: 7.6.2, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.3.0
cifarsmall
1
 0 : compute_capability = 750, cudnn_half = 1, GPU: GeForce RTX 2080 SUPER
net.optimized_memory = 0
mini_batch = 128, batch = 128, time_steps = 1, train = 1
   layer   filters  size/strd(dil)      input                output
   0 conv     32       3 x 3/ 1     28 x  28 x   3 ->   28 x  28 x  32 0.001 BF
   1 max                2x 2/ 2     28 x  28 x  32 ->   14 x  14 x  32 0.000 BF
   2 conv     16       1 x 1/ 1     14 x  14 x  32 ->   14 x  14 x  16 0.000 BF
   3 conv     64       3 x 3/ 1     14 x  14 x  16 ->   14 x  14 x  64 0.004 BF
   4 max                2x 2/ 2     14 x  14 x  64 ->    7 x   7 x  64 0.000 BF
   5 conv     32       1 x 1/ 1      7 x   7 x  64 ->    7 x   7 x  32 0.000 BF
   6 conv    128       3 x 3/ 1      7 x   7 x  32 ->    7 x   7 x 128 0.004 BF
   7 conv     64       1 x 1/ 1      7 x   7 x 128 ->    7 x   7 x  64 0.001 BF
   8 conv     10       1 x 1/ 1      7 x   7 x  64 ->    7 x   7 x  10 0.000 BF
   9 avg                             7 x   7 x  10 ->     10
  10 softmax                                          10
Total BFLOPS 0.010
avg_outputs = 6165
 Allocate additional workspace_size = 30.02 MB
Learning Rate: 0.1, Momentum: 0.9, Decay: 0.0005
50000
 If error occurs - run training with flag: -dont_show
 Create 32 permanent cpu-threads
Loaded: 0.554000 seconds
CUDA status Error: file: D:/YOLOv4/darknet-master_GPU/src/convolutional_kernels.cu : cuda_convert_f32_to_f16() : line: 138 : build time: Jul 16 2020 - 14:00:25

 CUDA Error: invalid device function

**Comment by NEHA2963** [comment]
I have the same issue, is there any fix?
